magento/magento2
# Adding created_at, status and grand_total filters to customer Orders query and fixing multiple filters failure

### Description (*)

Added created_at, status, and grand_total filters to customer Orders query
Fixing multiple filters bug in this query, when adding one more filter in the array for filterGroupBuilder addFilters it doesn´t apply all filters

### Related Pull Requests


### Fixed Issues (if relevant)

1. Fixes magento/magento2#38392

### Manual testing scenarios (*)

1. Create an order with a customer
2. With the same customer, generate a customer token and call the customer Orders query
3. Create multiple orders to test filters
4. Use two or more filters to check if multiple are being applied

### Questions or comments

I did not find a unit test for this class, does it exist?

I used the same names from the output object to maintain the pattern

I accept any suggestions or critics, doing it you are helping me!

### Contribution checklist (*)
